### PR TITLE
feat/32 input component

### PR DESCRIPTION
**Pull request by Trent1900**
### Descriptions

- Link to UI design (if possible):
image

### Provide screenshots or videos that the function is working.
image

### Provide screenshots of no warning, or error in the console.

- [x] All unit tests passing and coverage of testing is more than 90% unless special needs.
- [x] No merge conflict with the `main` branch, make sure your code is up to date with `main`.
- [x] Check PR title is aligned with your ticket.
- [x] Check your app will run without crashes.
- [x] Check if there are hard-coded values.
- [x] No `//eslint-disable-next-line no-unused-vars`, unless otherwise specified.
- [x] Check you are using `camelCase && PascalCase` correctly.
- [x] Remove unused comments.

### CODE REVIEW

File: src/components/Input/Search.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        Search for music, album and artist
+      </label>
+      <div className="pointer-events-none absolute inset-y-0 left-0 hidden items-center pl-3 sm:flex">
+        <AiOutlineSearch className="h-4 w-4 text-light-400"></AiOutlineSearch>
+      </div>
+      <input
+        id="searchInput"
+        placeholder="Search for music, album and artist"
+        className="hidden h-9 w-full rounded-full border border-light-100 bg-black/75 py-2 px-3 pl-9 text-xl text-light-400 placeholder:text-light-100
+          hover:border-light-400 hover:placeholder:text-light-400 focus:border-light-400 focus:outline-none focus:placeholder:text-transparent sm:flex"
```
Comment (Trent1900) [inline src/components/Input/Search.tsx:28]: comment from  **Double-Twelve**
should be `py-2` `pl-9` focus:placeholder:text-transparent
should remove `items-center`
should add placeholder:`text-light-100` `hover:border-light-400` `hover:placeholder:text-light-400`

File: src/components/Input/Search.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <form className="relative flex grow items-center justify-end" onSubmit={onSubmit}>
+      <label htmlFor="searchInput" className="sr-only">
+        Search for music, album and artist
+      </label>
+      <div className="pointer-events-none absolute inset-y-0 left-0 hidden items-center pl-3 sm:flex">
+        <AiOutlineSearch className="h-4 w-4 text-light-400"></AiOutlineSearch>
+      </div>
+      <input
+        id="searchInput"
+        placeholder="Search for music, album and artist"
```
Comment (tatoMa) [inline src/components/Input/Search.tsx:26]: You sure the placeholder will always like this?

**Review by tatoMa — COMMENTED** [review]
So you have a Button component inside of the UI folder, also have a Search component inside of the Input folder?

File: src/components/Input/Search.tsx (hunk 2 above)
Comment (Trent1900) [reply src/components/Input/Search.tsx]: thanks for remindering, change to props with default value : "Search for music, album and artist"

**Comment by Trent1900** [comment]
> So you have a Button component inside of the UI folder, also have a Search component inside of the Input folder?

I want to check the ui folder to be "button" folder, ui is too general and seems no other developers are using the folder

**Review by Double-Twelve — APPROVED** [review]

**Pull request merged**